tests/sentry/db/postgres/lookups/test_array_element_contains.py

```diff
@@ -0,0 +1,30 @@
+from unittest.mock import Mock, patch
+
+from sentry.db.postgres.lookups import ArrayElementContains
+from sentry.testutils.cases import TestCase
+
+
+class TestArrayElementContains(TestCase):
+    def setUp(self) -> None:
+        self.compiler = Mock()
+        self.connection = Mock()
```

Comment:
no reason to assign these to `self` -- they are only used in one place

and actually I don't think there's a reason to write a deprecated TestCase class either -- write a pytest function test please!